### PR TITLE
add mdonatello

### PR DESCRIPTION
**Pull request by talagayev**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by talagayev** [comment]
@conda-forge/help-python, the package is ready for review.

The linter complains "Please do not delete the example recipe found in recipes/example/recipe.yaml.", but it was not touched in this PR. 

The problem is tracked in https://github.com/conda-forge/staged-recipes/issues/27242 .

File: recipes/mdonatello/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mdonatello-{{ version }}.tar.gz
+  sha256: 719784cb4fc8a64a4df1aef484eac7e6220a142ea2b39db90e87056459b097d8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  
+requirements:
+  host:
```
Comment (synapticarbors) [inline recipes/mdonatello/meta.yaml:18]: Is `versioningit` needed here?

File: recipes/mdonatello/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - python >=3.9
+    - pip
+    - setuptools >=61.2
+  run:
+    - python >=3.9
+    - MDAnalysis >=2.0.0
```
Comment (synapticarbors) [inline recipes/mdonatello/meta.yaml:24]: `- mdanalysis >=2.0.0`

File: recipes/mdonatello/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    - jupyter
```
Comment (synapticarbors) [inline recipes/mdonatello/meta.yaml:25]: Should this be `jupyter` or:

```
- notebook
```
?

File: recipes/mdonatello/meta.yaml (hunk 1 above)
Comment (talagayev) [reply recipes/mdonatello/meta.yaml:18]: Yes my bad, adjusted it.

File: recipes/mdonatello/meta.yaml (hunk 3 above)
Comment (talagayev) [reply recipes/mdonatello/meta.yaml]: notebook would be enough for this package, adjusted it.

**Comment by talagayev** [comment]
@synapticarbors I adjusted all your suggestions, my bad for the mistakes in the packages, thanks for taking a look at them :crying_cat_face:

**Pull request merged**